wakatime/riscv64.patch

```diff
@@ -0,0 +1,36 @@
+--- PKGBUILD
++++ PKGBUILD
+@@ -14,12 +14,21 @@ license=('BSD')
+ depends=('glibc')
+ makedepends=('git' 'go')
+ checkdepends=('svn')
+-source=("${pkgname}-cli::git+https://github.com/${pkgname}/${pkgname}-cli#tag=v${pkgver}")
+-sha256sums=('SKIP')
++source=("${pkgname}-cli::git+https://github.com/${pkgname}/${pkgname}-cli#tag=v${pkgver}"
++        "main_test-arch.patch::https://github.com/wakatime/wakatime-cli/pull/645.patch")
```

Comment:
The patch name looks general enough that may conflict with others when `SRCDEST` is set. Please prefix it with something like `$pkgname-`.